docs/api/index.rst

```diff
@@ -11,6 +11,17 @@ API
 Changelog
 ---------
 
+1.13 (2016-10-18)
+'''''''''''''''''
+
+- Add support to JSON ``patch`` format when using 
+  ``Content-Type: application/json-patch+json`` (as in RFC 6902).
+- when using JSON ``patch``, changes should be described as a list of operations
+  (e.g ``[{"op": "add", "path": "/data/my_field", "value": "my_value"}]``).
+- On JSON ``patch`` operations, permissions are trated as a dictionary with no values.
+  On these operations, the ``value`` field can be omitted.
+  (e.g ``[{"op": "add", "path": "/permissions/read/system.Everyone"}]``).
```

Comment:
Right, when he said "a link" I think he meant a link that points to the API docs, not a "live" URL or command to demonstrate the PATCH functionality. So pointing to the API docs sounds fine to me, since this is just an index.